content/Services/RHOBS/design.md

```diff
@@ -0,0 +1,196 @@
+
+> NOTE: This document will focus on the metric signal. Logging is only partially deployed in certain places but is not yet a critical dependency on anything. It is also deployed in logical and network separation to the metric backend, so it should not affect the metric system; thus, all signals can be described separately. Tracing is in heavy progress as well.
+
+## Metrics
+
+### Architecture
+
+The current high-level architecture of the production service looks as follows:
+
+![](../../assets/rhobs-design1.png)
+
+Legend:
+
+* Blue background: Managed by RHOBS.
+* Grey background: Third-party managed software.
+* Orange arrows: Metric ingestion.
+* Red arrows: Firing alert notifications.
+* Green arrow: Metric / Rules / Alerts Read API traffic.
+* Black lines: Backbone communication (e.g. with object storage or authZ/authN with SSO).
+
+The RHOBs design is based on multiple shards that represent isolated [RHOBS-specific Observatorium](https://github.com/rhobs/configuration) installations. Shards are currently deployed to two clusters, but the design is prepared to have more clusters in different regions to lower the damage radius for regional incidents.
+
+All clusters are [the OSD](https://docs.openshift.com/dedicated/welcome/index.html), multi-availability zone clusters scheduled using AppSRE's AppInterface declarative GitOps API. See (behind VPN):
+
+* [telemeter-prod-01](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/openshift/telemeter-prod-01/cluster.yml)
+* [rhobsp02ue1](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/openshift/rhobsp02ue1/cluster.yml)
+
+Each Observatorium installation represents RHOBS "shard" and uses multiple Kubernetes namespaces. Typically single shard is limited by the single cluster capacity (amount of the nodes it can handle--usually around 200 for OSD), so we keep a single shard in a single cluster. The only exception is `Telemeter` deployment, which for historical reasons is a separate ("hard") tenant deployed next to `MST` in one cluster. `Telemeter` tenant is used for [OpenShift remote health monitoring](https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.5/html/monitoring_openshift_container_storage/remote_health_monitoring).
+
+> NOTE: Telemeter was the first responsibility of our service team in the past. We build RHOBS in a way that Telemeter can be a single-region hard tenant. It is isolated because it has different characteristics as typical observability data. It contains very cardinal data with only ~5m sample interval. There is no alerting, only analytic PromQL queries and integration with analytics tools like [obslytics](https://github.com/thanos-community/obslytics). We will focus on `MST` in this design as this is what will be used by the majority of the users.
```

Comment:
```suggestion
> NOTE: Telemeter was the first responsibility of our service team in the past. We built RHOBS in a way that Telemeter can be a single-region hard tenant. It is isolated because it has different characteristics than typical observability data. It contains very cardinal data with only ~5m sample interval. There is no alerting, only analytic PromQL queries and integration with analytics tools like [obslytics](https://github.com/thanos-community/obslytics). We will focus on `MST` in this design as this is what will be used by the majority of the users.
```